### PR TITLE
Add a TOML tab to HttpPushConsumer#show

### PR DESCRIPTION
**Pull request by acco**
CleanShot 2024-09-27 at 16 58 58@2x

### CODE REVIEW

**Comment by acco** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/324?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#324** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/324?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#323** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/323?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @acco and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by acco** [comment]
CleanShot 2024-09-27 at 16 58 58@2x

File: lib/sequin/consumers/http_push_consumer.ex [hunk 1]
```diff
@@ -57,6 +57,28 @@ defmodule Sequin.Consumers.HttpPushConsumer do
...
+        end
+
+      map = %{
+        ack_wait_ms: consumer.ack_wait_ms,
+        max_ack_pending: consumer.max_ack_pending,
+        max_deliver: consumer.max_deliver,
+        max_waiting: consumer.max_waiting,
+        message_kinds: messages_kind,
+        name: consumer.name,
+        http_endpoint_id: consumer.http_endpoint_id
```
Comment (thisisgoldman) [inline lib/sequin/consumers/http_push_consumer.ex:75]: Should this be `endpoint_name` -- to keep it all names as opposed to id

**Review by thisisgoldman — APPROVED** [review]
I'm happy we're making this happen!

Instead of a tab - perhaps this is just another popover button in the Configuration card? I don't know if it needs to take up that much real-estate since it's static and the information is already in that card:

CleanShot 2024-09-27 at 18 01 05@2x

Maybe it's just called sequin.config.toml or the name of the file we expect to process.

I think it makes sense for each resource to have a toml in th console for now - and then when we have projects / environments, a dev can export a toml for all the resources in the project. Either way, it's a next iteration item.

File: lib/sequin/consumers/http_push_consumer.ex (hunk 1 above)
Comment (acco) [reply lib/sequin/consumers/http_push_consumer.ex:75]: `http_endpoint` sounds right - good call

**Pull request closed without merging**